packages/rest/src/__tests__/acceptance/validation/validation.acceptance.ts

```diff
@@ -29,8 +32,14 @@ describe('Validation at REST level', () => {
     @property({required: true})
     name: string;
 
-    @property({required: false})
-    description?: string;
+    // We have to add `type: 'string'` as `string | null` removes TypeScript
+    // design:type reflection
+    @property({required: false, jsonSchema: {nullable: true, type: 'string'}})
```

Comment:
I find this suspicious. Why is `type: 'string'` set at `jsonSchema` level? How is juggler and connectors going to know that the property type is string?

I would expect the following property definition:

```suggestion
    @property({required: false, type: 'string', jsonSchema: {nullable: true}})
```